src/compressed_tensors/quantization/quant_args.py

```diff
@@ -51,6 +52,20 @@ class QuantizationStrategy(str, Enum):
     TOKEN = "token"
 
 
+class ActivationOrderingStrategy(str, Enum):
+    """
+    Enum storing strategies for activation ordering
+
+    Weight := only reorder weight, not groups (default)
+    Grouped := reorder groups
+    Off := do not reorder by activations
+    """
+
+    WEIGHT = "weight"
+    GROUP = "group"
+    OFF = "off"
```

Comment:
if off means not to use actorder, 
then you can just set the value to None, no need to populate here

Reply:
Yes

Reply:
Ok then I think we can set that to None, we might not want to allocate the variable

Reply:
The model currently accepts True and False. Adding the `None` type would be redundant to `False`

```python3
actorder: Union[ActivationOrderingStrategy, bool] = ActivationOrderingStrategy.OFF
```python3
```
def validate_actorder(cls, value) -> ActivationOrderingStrategy:
        if isinstance(value, bool):
            # default to weight if True
            return (
                ActivationOrderingStrategy.WEIGHT
                if value
                else cls.model_fields["actorder"].default
```

Reply:
If you have weight and group, you dont need the bool.

Reply:
Yes, but it's nice to support for those who want to use the same api as autogptq